_posts/2024-10-21-Session-Spotlight-OpenSearchCon-Keynote.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title:  'Session Spotlight: Keynote Address at OpenSearchCon North America 2024'
+authors:
+ - mlutton
+date: 2024-10-21
+categories:
+ - community-updates
+meta_keywords: OpenSearchCon, OpenSearch Project, Adobe, Linux Foundation, GenAI, Vector Search, OpenSearch Software Foundation
+meta_description: Watch the OpenSearchCon North America 2024 keynote address and hear from featured speakers from OpenSearch, Adobe, and the Linux Foundation as they discuss uses cases along with the project’s history and future.
+excerpt: 'The Opening Keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation. '
+has_math: false
+has_science_table: false
+youtubeID: WPVGfWoR0B8
+---
+
+Whether or not you were able to join us in San Francisco for OpenSearchCon North America 2024 now is a fantastic time to catch up with recordings of the sessions. In our first Session Spotlight we’re bringing you back to the kickoff of the conference on September 24th, 2024. 
+
+The Opening Keynote for OpenSearchCon North America 2024 was given by Eli Fisher, Senior Manager of Project Management at AWS for OpenSearch, alongside two very special guests: Anupam Chaturvedi, Engineering Manager for Acrobat GenAI Services at Adobe, and Jim Zemlin, Executive Director of the Linux Foundation. 
+
+{% include embed/youtube.html id=page.youtubeID %}
+
+In this keynote Eli Fisher outlines the history of the OpenSearch Project from inception to this momentous occasion of joining the [Linux Foundation]( ) as the [OpenSearch Software Foundation](http://foundation.opensearch.org) in September 2024. Anupam Chaturvedi also dives in to the exciting use case of GenAI-assisted search in PDF documents at [Adobe](https://www.adobe.com/acrobat/generative-ai-pdf.html), enabled by the powerful vector search capabilities of OpenSearch. And finally, Jim Zemlin makes the case for the bright future of open source software and highlights the strength of partnering with projects like OpenSearch. 
+
```

Comment:
Above: Ensure that LF link is added.

Reply:
Hi @natebower , 
Changes have been made!